### ISSUE TITLE
is css-modules supported with jspm

### ISSUE THREAD

**Issue by jithine** [css-modules/jspm-demo#7]
jspm-loader-css-modules is broken with latest jspm release `v0.16.13`. Is jspm css-modules project abandoned ?

**Comment by nomadtechie** [comment]
I saw this same issue as well, I installed this module and it worked? https://github.com/systemjs/plugin-css

**Comment by jithine** [comment]
I am also now using `plugin-css` but miss css-modules functionality.

**Comment by nomadtechie** [comment]
Hummm. Can you elaborate @jithin1987 , even with the plugin-css I am not able to export classes from the css file?

**Comment by jithine** [comment]
`plugin-css` works on plain css files. No fancy processing on top of it.

**Comment by geelen** [comment]
Yep, that's right @jithin1987 I'm working on fixing bundling in `jspm-loader-css` now as well
